adapters/bidmachine/bidmachine_test.go

```diff
@@ -0,0 +1,89 @@
+package bidmachine
+
+import (
+	"math"
+	"net/http"
+	"testing"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderBidmachine, config.Adapter{
+		Endpoint: "http://{{.Host}}.example.com/prebid"})
+
+	if buildErr != nil {
+		t.Fatalf("Builder returned unexpected error %v", buildErr)
+	}
+
+	adapterstest.RunJSONBidderTest(t, "bidmachinetest", bidder)
+}
+
+func makeBidRequest() *openrtb.BidRequest {
+	request := &openrtb.BidRequest{
+		ID: "test-req-id-0",
+		Imp: []openrtb.Imp{
+			{
+				ID: "test-imp-id-0",
+				Banner: &openrtb.Banner{
+					Format: []openrtb.Format{
+						{
+							W: 320,
+							H: 50,
+						},
+					},
+				},
+				Ext: []byte(`{"bidder":{"seller_id":"1", "host": "api-eu", "path":"auction/rtb/v2`),
+			},
+		},
+	}
+	return request
+}
+
+func TestMakeRequests(t *testing.T) {
```

Comment:
Could you please represent these tests in the json framework format? You can use the regex comparison type to account for differences in Go versions. Out of curiosity, which versions of Go are you seeing the change with?